app/views/unattended/provisioning_templates/snippet/redhat_register.erb

```diff
@@ -292,3 +292,4 @@ snippet: true
     echo "No activation key found: Not registering"
   <% end %>
 <% end %>
+
```

Comment:
I've managed to generate the snapshots locally, looks like the only change is actually due to this extra newline. If you remove it from the commit snapshots should pass.
```suggestion
```

Reply:
Thank you @tbrisker.  Maybe I did that wrong.  I see now that I could have committed your suggestion vs. pushing my own attempt to make that change.